nilearn/interfaces/bids.py

```diff
@@ -158,3 +168,324 @@ def parse_bids_filename(img_path):
         else:
             reference[field] = None
     return reference
+
+
+def save_glm_to_bids(
```

Comment:
If this is going to be a public function, could you add it to the API doc?

Reply:
Done in [d85897e](https://github.com/nilearn/nilearn/pull/2715/commits/d85897ec2e8eebb0aebaa25aaf4f1576b92dff62). Sorry, I forgot to follow up in this thread.